src/data_hub/dataframe.py

```diff
@@ -0,0 +1,219 @@
+"""
+This module defines a class called DataFrame that is a drop-in
+replacement for a pandas DataFrame, but also allows automatic
+updating of Deep Origin databases.
+"""
+
+from datetime import datetime, timezone
+from typing import Optional
+
+import humanize
+import pandas as pd
+from deeporigin.data_hub import api
+from deeporigin.utils import (
+    DatabaseReturnType,
+    IDFormat,
+    construct_resource_url,
+)
+
+
+class DataFrame(pd.DataFrame):
+    """A subclass of pandas DataFrame that allows for automatic updates to Deep Origin databases. This can be used as a drop-in replacement for a pandas DataFrame, and should support all methods a pandas DataFrame supports.
```

Comment:
I suggest removing "automatic" since its no longer the default behavior

Reply:
fixed